active/NavigationView/NavigationViewTemplateSettings/PaneTitle.md

```diff
@@ -0,0 +1,45 @@
+# Background
+PaneTitle is added to NavigationView in RS4, but it's a String. Since RS5, we introduced PaneHeader(UIElement) to deprecate PaneTitle. If both PaneTitle and PaneHeader are provided by customer, PaneHeader is used.
+
+This API update addes PaneTitle [Template-Settings](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) to the NavigationView control.
+
+# Description
+Provides calculated PaneTitle that can be referenced as TemplatedParent sources when defining templates for a NavigationView control. Not intended for general use.
+
+# Examples
+
+```
+[webhosthidden]
+unsealed runtimeclass NavigationViewTemplateSettings : Windows.UI.Xaml.DependencyObject

```

Comment:
This isn't an Example, this is the API, and should be in the API Details section.  The "sample usage" from the Remarks section should be brought up here.